wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/httpcamera/Robot.java

```diff
@@ -27,14 +27,17 @@ public void robotInit() {
     m_visionThread =
         new Thread(
             () -> {
-              // Get the Axis camera from CameraServer
-              AxisCamera camera = CameraServer.addAxisCamera("axis-camera.local");
+              // Create an HTTP camera
+              HttpCamera camera =
+                  new HttpCamera("HTTP Camera", "http://10.x.y.11/video/stream.mjpg");
+              // Start capturing images
+              CameraServer.startAutomaticCapture(camera);
```

Comment:
Should add comment that the address will need to be modified to have the correct team number and that the path will depend on the source.